Home/Home 2.1/hello.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+struct UserStat: Codable {
+    var name: String
+    var count: Int
+}
+
+class UserStatsManager {
+    private var userStats: [String: UserStat] = [:]
+    private let filename = "user_stats.txt"
```

Comment:
Looks good while we can apply comments related to constants and filename as constructor parameter.

Reply:
Thanks for the comment. Good point.